src/components/AuthorBlogPostCard/index.tsx

```
@@ -0,0 +1,79 @@
+import { GatsbyImage, getImage } from 'gatsby-plugin-image';
+import { Link } from 'gatsby';
+import React from 'react';
+import ArrowRight2 from '../../svgs/arrow-right-2.svg';
+import Avatar from '../Avatar';
+import {
+    container,
+    popularArticlesImage,
+    articleCardHeader,
+    articleTag,
+    articleDateAndTime,
+    articleCardAuthors,
+    articleCardFooter,
+    dot,
+    authorInfo,
+} from './styles.module.less';
+
+interface AuthorBlogPostCardProps {
+    data: any;
+}
+
+const AuthorBlogPostCard = ({ data: blogPost }: AuthorBlogPostCardProps) => {
```

What about instead of having `blogPost?` all over we should check for that right at the start and return null. Maybe throw an error so we could catch it?

---

Actually blogPost should not be optional. I will remove the question mark.